samples/subsys/usb/uvc/boards/stm32n6570_dk_jpeg.overlay

```diff
@@ -0,0 +1,15 @@
+/*
+ * Copyright (c) 2025 STMicroelectronics
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/ {
+	chosen {
+		zephyr,videoenc = &jpeg;
+	};
+
+	uvc: uvc {
+		compatible = "zephyr,uvc-device";
+	};
+};
```

Comment:
Same as for other board specific overlay files, I do not see anything platform specific here, so it could be samples/subsys/usb/uvc/video_enc_jpeg.overlay but not a board DTS overlay.

Reply:
I think `jpeg` is the name of an STM32 JPEG encoder/decoder.

On devices that have a JPEG encoder only, this would be named `jpegenc` (or something more specific).
On devices that have a JPEG decoder only, this would be named `jpegdec` (or something more specific).

But *if* this happen then this could be modified at this time (maybe it's always `jpeg` that can do both encoding+decoding).